code/modules/reagents/reagent_containers/spray.dm

```diff
@@ -186,6 +186,19 @@
 	desc = "Gyaro brand spray tan. Do not spray near eyes or other orifices."
 	list_reagents = list(/datum/reagent/spraytan = 50)
 
+//bo spray
+/obj/item/reagent_containers/spray/spaceaxe
+	name = "nanotrasen shower-in-a-can"

```

Comment:
Nanotrasen is a proper noun and should be capitalized.

Reply:
You'll want to put a \improper at the beginning as well